docs/openai_api.ipynb

```diff
@@ -27,42 +29,96 @@
+      "[2024-10-28 02:02:37 TP0] Load weight begin. avail mem=47.27 GB\n",
+      "[2024-10-28 02:02:37 TP0] Ignore import error when loading sglang.srt.models.mllama. No module named 'transformers.models.mllama'\n",
+      "INFO 10-28 02:02:38 weight_utils.py:236] Using model weights format ['*.safetensors']\n",
+      "Loading safetensors checkpoint shards:   0% Completed | 0/4 [00:00<?, ?it/s]\n",
+      "Loading safetensors checkpoint shards:  25% Completed | 1/4 [00:00<00:01,  2.57it/s]\n",
+      "Loading safetensors checkpoint shards:  50% Completed | 2/4 [00:00<00:00,  2.45it/s]\n",
+      "Loading safetensors checkpoint shards:  75% Completed | 3/4 [00:00<00:00,  3.53it/s]\n",
+      "Loading safetensors checkpoint shards: 100% Completed | 4/4 [00:01<00:00,  2.98it/s]\n",
+      "Loading safetensors checkpoint shards: 100% Completed | 4/4 [00:01<00:00,  2.94it/s]\n",
+      "\n",
+      "[2024-10-28 02:02:40 TP0] Load weight end. type=LlamaForCausalLM, dtype=torch.bfloat16, avail mem=32.22 GB\n",
+      "[2024-10-28 02:02:40 TP0] Memory pool end. avail mem=4.60 GB\n",
+      "[2024-10-28 02:02:40 TP0] Capture cuda graph begin. This can take up to several minutes.\n",
+      "[2024-10-28 02:02:48 TP0] max_total_num_tokens=217512, max_prefill_tokens=16384, max_running_requests=2049, context_len=131072\n",
+      "[2024-10-28 02:02:48] INFO:     Started server process [1185529]\n",
+      "[2024-10-28 02:02:48] INFO:     Waiting for application startup.\n",
+      "[2024-10-28 02:02:48] INFO:     Application startup complete.\n",
+      "[2024-10-28 02:02:48] INFO:     Uvicorn running on http://0.0.0.0:30000 (Press CTRL+C to quit)\n",
+      "[2024-10-28 02:02:48] INFO:     127.0.0.1:47904 - \"GET /v1/models HTTP/1.1\" 200 OK\n"
      ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<strong style='color: #00008B;'>Server is ready. Proceeding with the next steps.</strong>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
     }
    ],
    "source": [
-    "from sglang.utils import execute_shell_command, wait_for_server, terminate_process\n",
+    "from sglang.utils import lauch_sglang_server, wait_for_server, terminate_process, highlight_text\n",
     "\n",
-    "server_process = execute_shell_command(\n",
-    "    \"\"\"\n",
-    "python -m sglang.launch_server --model-path meta-llama/Meta-Llama-3.1-8B-Instruct \\\n",
-    "--port 30000 --host 0.0.0.0 --log-level warning\n",
-    "\"\"\"\n",
+    "server_process = lauch_sglang_server(\n",
+    "    command=\"python -m sglang.launch_server --model-path meta-llama/Meta-Llama-3.1-8B-Instruct --port 30000 --host 0.0.0.0\"\n",
     ")\n",
     "\n",
     "wait_for_server(\"http://localhost:30000\")\n",
-    "print(\"Server is ready. Proceeding with the next steps.\")"
+    "\n",
+    "highlight_text(\"Server is ready. Proceeding with the next steps.\")"
```

Comment:
- mention that the cell will contain both the server output and notebook outout.
- mention that typically the server will run on a separate terminal, but here we combine the output into a single cell to demonstrate the usage better. Mention that the server output will be in gray while the notebook output will be highlighted.